src/quote/getQuoteAndBuyCoverInputs.ts

```diff
@@ -55,8 +82,9 @@ async function getQuoteAndBuyCoverInputs(
   coverAsset: CoverAsset,
   coverBuyerAddress: Address,
   slippage: number = DEFAULT_SLIPPAGE / SLIPPAGE_DENOMINATOR,
-  ipfsCid: string = '',
-  coverRouterUrl = process.env.COVER_ROUTER_URL!,
+  ipfsCidOrContent: string | IPFSContentForProductType[ProductTypes] = '',
+  coverRouterUrl = process.env.COVER_ROUTER_URL,
+  ipfsUploadUrl = process.env.IPFS_GATEWAY_URL,
```

Comment:
should we default this to our own IPFS upload API (I.e. hardcode it in as the default value) ? 

It will be better UX as they don't have to pass or set any env vars, it just works. but they still have the ability to override it if they wish